src/main/java/Duke.java

```diff
@@ -1,10 +1,161 @@
+        System.out.println("\t" + HORIZONTAL);
+    }
+
+    private static void printTasks(int countTasks, Task[] tasks) {
+        System.out.println("\t" + HORIZONTAL);
+        for (int i = 1; i <= countTasks; i++) {
+            if (tasks[i-1].getType() == "[D]") {
+                System.out.println("\t" + i + "." + tasks[i - 1].getType() + tasks[i - 1].getStatusIcon() + " " + tasks[i - 1].getDescription() + "(by:" +tasks[i - 1].getTime() +")");
+            }
+            if (tasks[i-1].getType() == "[E]") {
+                System.out.println("\t" + i + "." + tasks[i - 1].getType() + tasks[i - 1].getStatusIcon() + " " + tasks[i - 1].getDescription() + "(at:" +tasks[i - 1].getTime() +")");
+            }
+            else if (tasks[i-1].getType() == "[T]") {
+                System.out.println("\t" + i + "." + tasks[i - 1].getType() + tasks[i - 1].getStatusIcon() + " " + tasks[i - 1].getDescription() + tasks[i - 1].getTime());
+            }
+        }
+        System.out.println("\t" + HORIZONTAL);
+    }
+    private static int addTodos(int countTasks, Task[] tasks, String line) {
+        tasks[countTasks] = new ToDo(line);
+        countTasks++;
+        System.out.println("\tGot it. I've added this task: ");
+        System.out.println("\t"+tasks[countTasks - 1].getType() + tasks[countTasks - 1].getStatusIcon() + " " + tasks[countTasks - 1].getDescription());
+        System.out.print("\tNow you have " + countTasks + " task");
+        if (countTasks > 1)
+            System.out.print("s");
+        else System.out.print("");
+        System.out.println(" in the list.");
+        return countTasks;
+    }
+    private static int addEvents(int countTasks, Task[] tasks, String line, String time) {
+        tasks[countTasks] = new Event(line, time);
+        countTasks++;
+        System.out.println("\tGot it. I've added this task: ");
+        System.out.println("\t" + tasks[countTasks - 1].getType() + tasks[countTasks - 1].getStatusIcon() + " " + tasks[countTasks - 1].getDescription() + " at:" + tasks[countTasks - 1].getTime());
+        System.out.print("\tNow you have " + countTasks + " task");
+        if (countTasks > 1)
+            System.out.print("s");
+        else System.out.print("");
+        System.out.println(" in the list.");
+        return countTasks;
+    }
+    private static int addDeadlines(int countTasks, Task[] tasks, String line, String time) {
+        tasks[countTasks] = new Deadline(line, time);
+        countTasks++;
+        System.out.println("\tGot it. I've added this task: ");
+        System.out.println("\t" + tasks[countTasks - 1].getType() + tasks[countTasks - 1].getStatusIcon() + " " + tasks[countTasks - 1].getDescription() + " by:" + tasks[countTasks - 1].getTime());
+        System.out.print("\tNow you have " + countTasks + " task");
+        if (countTasks > 1)
+            System.out.print("s");
+        else System.out.print("");
```

Comment:
if-else blocks should ideally have brackets wrapping the text even if it is one statement only